nixos/modules/misc/version.nix

```diff
@@ -16,13 +16,13 @@ let
     ) + "\n";
 
   osReleaseContents = {
-    NAME = "NixOS";
-    ID = "nixos";
+    NAME = "${cfg.distroName}";
+    ID = "${cfg.distroId}";
     VERSION = "${cfg.release} (${cfg.codeName})";
     VERSION_CODENAME = toLower cfg.codeName;
     VERSION_ID = cfg.release;
     BUILD_ID = cfg.version;
-    PRETTY_NAME = "NixOS ${cfg.release} (${cfg.codeName})";
+    PRETTY_NAME = "${cfg.distroName} ${cfg.release} (${cfg.codeName})";
     LOGO = "nix-snowflake";
     HOME_URL = "https://nixos.org/";
     DOCUMENTATION_URL = "https://nixos.org/learn.html";
```

Comment:
Does links presumable don't apply for other distributions, do they?

Reply:
Maybe something like this would make sense? That way once changed, none of the links point back to official NixOS websites
```
    HOME_URL = if cfg.distroId == "nixos" then "https://nixos.org/" else "";
    DOCUMENTATION_URL = if cfg.distroId == "nixos" then "https://nixos.org/learn.html" else "";
    SUPPORT_URL = if cfg.distroId == "nixos" then "https://nixos.org/community.html" else "";
    BUG_REPORT_URL = if cfg.distroId == "nixos" then "https://github.com/NixOS/nixpkgs/issues" else "";
```

Reply:
I agree